pkg/accprovisioninput/apis/aci.ctrl/v1alpha1/types.go

```diff
@@ -51,6 +51,7 @@ type SyncLoginSpec struct {
 
 type NetConfigSpec struct {
 	InterfaceMTU           int  `json:"interface_mtu,omitempty"`
+	IntMtuHeadroom         int  `json:"int-mtu-headroom,omitempty"`
```

Comment:
1. i think the "-" should be "_"

2. for the sake of consistency can we call this "interface_mtu_headroom" and "InterfaceMtuHeadroom"?